engine-rest/engine-rest-openapi/src/main/templates/models/org/camunda/bpm/engine/rest/dto/identity/PasswordPolicyRuleDto.ftl

```diff
@@ -0,0 +1,17 @@
+<@lib.dto desc = "Describes a rule of a password policy.">
+
+    <@lib.property
+        name = "placeholder"
+        type = "string"
+        desc = "A placeholder string that can be used to display an internationalized message to the user."
```

Comment:
This is taken from the documentation, but I think it is a little unspecific. Could we put something more descriptive here and also for the next property?